satellite/satellitedb/dbx/satellitedb.dbx

```diff
@@ -564,3 +564,48 @@ read one (
     where reset_password_token.owner_id = ?
 )
 delete reset_password_token ( where reset_password_token.secret = ? )
+
+
+//--- offer table ---//
+
+model offer (
+	key id
+	
+	field id blob
+	field name text ( updatable )
```

Comment:
should all these fields be updatable? seems odd the name of the offer would change

Reply:
That's a good point. I will check in with the business to see if they actually have this need

Reply:
Note that it’s fine to leave off the “updatable” attribute until you need it. It doesn’t affect the SQL modeling at all, and we can change it later without requiring a migration or anything.